src/MDFRobotoFontLoader.h

```diff
@@ -83,4 +83,15 @@
  */
 - (nonnull UIFont *)boldItalicFontOfSize:(CGFloat)fontSize;
 
+/** Returns YES if the font name represents what we consider to be an "italic" font. */
++ (BOOL)isItalicFontName:(nonnull NSString *)fontName;
```

Comment:
These 'name'-oriented APIs are a bit inconsistent with the rest. I don't see any internal usage of `isItalicFontName:` and `isBoldFontName:`, how about omit these two?